runconfig/hostconfig.go

```diff
@@ -202,6 +232,7 @@ func (c *CapList) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Len returns the number of specific lxc configuration
```

Comment:
Hum on this one the structure is ``LxcConfig``, but yeah, the comment on the ``CapList`` were wrong, I've updated it.